helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.chart }}
+  labels:
+    app: {{ .Values.name }}
+    giantswarm.io/service-type: "managed"
+rules:
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - "*"
+  - apiGroups:
+      - application.giantswarm.io
+    resources:
+      - charts
+    verbs:
+      - "*"
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    app: {{ .Values.name }}
+    giantswarm.io/service-type: "managed"
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - "*"
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - apps
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - apps/status
+  verbs:
+    - create
+    - patch
+    - update
+- apiGroups:
+  - application.giantswarm.io
+  resources:
+  - apps
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - appcatalogs
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - apps
```

Comment:
This looks strange to me. `apps` is in the role multiple times. Any dupes need to be removed.